Multiple Fixes

cyberhiker opened the pull request:
Attempting to get this working with Microsoft SQL Server.  Debugger is no longer supported and I couldn't build the gem, I don't know enough about Ruby to fix all that though.  One of the things that was an issue was that the ActiveRecord stuff has been deprecated.  I muddled through getting the latest version to work.  

Again, this is very dirty and probably only partially worked.  It did what I needed it to do, pull it in if you like or discard if you don't.

anlek commented:
Hey @cyberhiker,
Thank you so much for contributing!
I'm unclear what you're trying to fix, the current pull request just adds a few files to the .gitignore system. In my opinion they don’t actually belong there.
Let me know if I'm missing anything. Thanks!

cyberhiker commented:
My bust, you aren't missing anything, I'll close this one out and try again.  It's my second pull request, so please be patient with me.

Having said that, I had a heck of a time getting this to work with Ruby 2.1.  Apparently, pry-plus needs debugger 1.6.8 which isn't supported on Ruby 2.1.  So I changed the gemspec to pry-plus-byebug and the bundler and gem build began working, but I have no idea if that piece of it is used during the translation or processing code though.

I updated some of the SQL statements inside sql_connection.rb with brackets to be able to support SQL Server.

anlek commented:
Oh, thanks! If it's possible to commit the Ruby 2.1 fix, I'd love merge that one right off the bat! :)
The other one I'll review and get back to you on my thoughts.
I don't have a SQLServer setup, so I can't test it :(
